### ISSUE TITLE
pluralization problem

### ISSUE THREAD

**Issue by LutzStrobel** [i18next/ng-i18next#84]
We have problems, maybe because of a missing understanding, to get a correct translations for a given pluralization case (see below). We expect to 
get : 
   `"Wochenende!!!!"`
for the html: 
   `<p>{{'days.rest' | i18next:{count:0} }}</p>`,
but we get always:
`"Noch 0 Tage bis zum Wochenende!"`.

Does anybody has a hint whats wrong with our code?
Thanx
Lutz
-----------------------------------------------------------------------------------------------
We use the following translation resources:

//DE
  	"days":{
	  "rest": "Noch  __count__  Tag bis zum Wochenende.",
	  "rest_plural": "Noch __count__ Tage bis zum Wochenende!",
	  "rest_plural_0": "Wochenende!!!!"
	},
//EN
	"days":{
	  "rest": "Still __count__  day until weekend.",
	  "rest_plural": "Still __count__ days until weekend!",
	  "rest_plural_0": "Weekend, work is out!!!!"
	},

and within the .../filter/passingOptions.html

		<p>{{'days.rest' | i18next:{count:0} }}</p>
		<p>{{'days.rest' | i18next:{count:1} }}</p>
		<p>{{'days.rest' | i18next:{count:4} }}</p>
--------------------------------------------------------------------

**Comment by archer96** [comment]
It seems neither `rest_plural_0` nor `rest_plural_4` work. The options are passed to `i18next` but it still doesn't use `rest_plural_0`...

@jamuhl Could you have a look on this, please? `ng-i18next` passes `count:0` to `i18next.t()` but doesn't get the correct translation.

Regards,
Andre

**Comment by jamuhl** [comment]
there is nothing like resolving `count: 0` to some `.._plural_0`

the `plural_number` are used for languages that know more than one pluralization form.

**Comment by archer96** [comment]
What about 'simple plural' here: http://i18next.com/pages/doc_features.html
It's quite confusing because reading the documentation makes me think that something like `myword_plural_99' would work. Could you then add a note, please?

Regards,
Andre

**Comment by jamuhl** [comment]
yes...it's really about languages with multiple forms for few, many,...(or with only one form)...

will try to make it more clear for next release that the multiple plurals is about languages with multiple plurals and not about different plurals for eg. `en`

i thought about that feature before....would need something like: `myKey_plural_n>=10&&n<100` but gave up on that idea...as it would really need to much logic to evaluate this and would break on gettext.

basically one could archive this by using a combination of `context` and `plurals` but still a lot of rules to define.

**Comment by archer96** [comment]
ok. Thanks for that info :)
